pmm-tests/minikube_operators_setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Prepare a set of base64 encoded values and non encoded for user and pass with administrator privileges to pmm-server (DBaaS)
+PMM_USER='admin';
+PMM_PASS='admin';
+OPERATOR_VERSION=$1;
+
+PMM_USER_B64="$(echo -n "${PMM_USER}" | base64)";
+PMM_PASS_B64="$(echo -n "${PMM_PASS}" | base64)";
+
+# Install the PXC operator
+curl -sSf -m 30 https://raw.githubusercontent.com/percona/percona-xtradb-cluster-operator/pmm-branch/deploy/bundle.yaml | minikube kubectl -- apply -f -
+curl -sSf -m 30 https://raw.githubusercontent.com/percona/percona-xtradb-cluster-operator/pmm-branch/deploy/secrets.yaml | sed "s/pmmserver:.*=/pmmserver: ${PMM_PASS_B64}/g" | minikube kubectl -- apply -f -
```

Comment:
Consider wrapping this line for readability

Reply:
yes done